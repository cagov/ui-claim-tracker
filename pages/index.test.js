//import MyApp from "./_app";
//import React from "react";
//import { shallow } from "enzyme";
//
//describe("<MyApp />", () => {
//  it("renders application", () => {
//    const wrapper = shallow(<MyApp />);
//
//    expect(wrapper).toMatchSnapshot();
//  });
//});

import React from 'react'
import renderer from 'react-test-renderer'
import {render, screen} from '@testing-library/react'
import '@testing-library/jest-dom'
import Index from '../pages/index'

describe("Exemplar-react-test renderer Snapshot test", () => {
  it('renders homepage unchanged', () => {
    const tree = renderer.create(<Index />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})


describe("Example react testing-library Test", () => {
  it('has NextJS welcome text', () => {
    render(<Index />)
    expect(screen.queryByText("Welcome to")).toBeInTheDocument()
  })
})
