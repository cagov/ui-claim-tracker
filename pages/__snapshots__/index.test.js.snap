// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exemplar-react-test renderer Snapshot test renders homepage unchanged 1`] = `
<div>
  <main>
    <h1>
      Welcome to 
      <a
        href="https://nextjs.org"
      >
        Next.js!
      </a>
    </h1>
    <p>
      Get started by editing
       
      <code>
        pages/index.js
      </code>
    </p>
    <div>
      <a
        href="https://nextjs.org/docs"
      >
        <h3>
          Documentation →
        </h3>
        <p>
          Find in-depth information about Next.js features and API.
        </p>
      </a>
      <a
        href="https://nextjs.org/learn"
      >
        <h3>
          Learn →
        </h3>
        <p>
          Learn about Next.js in an interactive course with quizzes!
        </p>
      </a>
      <a
        href="https://github.com/vercel/next.js/tree/master/examples"
      >
        <h3>
          Examples →
        </h3>
        <p>
          Discover and deploy boilerplate example Next.js projects.
        </p>
      </a>
      <a
        href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      >
        <h3>
          Deploy →
        </h3>
        <p>
          Instantly deploy your Next.js site to a public URL with Vercel.
        </p>
      </a>
    </div>
  </main>
  <footer>
    <a
      href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      Powered by
       
      <img
        alt="Vercel Logo"
        src="/vercel.svg"
      />
    </a>
  </footer>
</div>
`;
